lottie/src/main/java/com/airbnb/lottie/LottieAnimationView.java

```diff
@@ -318,7 +317,7 @@ protected void onVisibilityChanged(@NonNull View changedView, int visibility) {
 
   @Override protected void onAttachedToWindow() {
     super.onAttachedToWindow();
-    if (autoPlay || wasAnimatingWhenDetached) {
+    if (!isInEditMode() && autoPlay || wasAnimatingWhenDetached) {
```

Comment:
I don't think this is right. You want parens around the two ORs

Reply:
Thanks a lot.  I had fix this. Please review again.